developers/contributor-guide/weaviate-core/tests.md

```diff
@@ -120,7 +120,7 @@ Note that while slow tests are optional on the integration test runner script, t
 To mark an integration test as "slow" simply use the `integrationTestSlow` build tag, instead of the `integrationTest` tag.
 
 ### Journey tests
-As outline in the Philosophy, journey tests require the application to be compiled as well as all backing services to be running. We have a convenience script available which starts all required services in `docker-compose` and runs the tests against it.
+As outline in the Philosophy, journey tests require the application to be compiled as well as all backing services to be running. We have a convenience script available which starts all required services in `docker compose` and runs the tests against it.
```

Comment:
This is also a product name. It should be capitalized, although without the dash. Nicely spotted!